src/sentry/performance_issues/detectors/consecutive_db_detector.py

```diff
@@ -271,8 +271,8 @@ def is_event_eligible(cls, event: dict[str, Any], project: Project | None = None
 
 def contains_complete_query(span: Span, is_source: bool | None = False) -> bool:
     # Remove the truncation check from the n_plus_one db detector.
-    query = span.get("description", None)
+    query = span.get("description")
     if is_source and query:
         return True
     else:
-        return query and not query.endswith("...")
+        return bool(query and not query.endswith("..."))
```

Comment:
slightly faster would be `return query is not None and ...`

Reply:
although idk what it does for `"query": ""`

Reply:
yeah i don't 100% trust the annotations, if there's `0` passed here i'd rather the type annotations be silently wrong than breaking prod